.github/workflows/github-dispatches.yaml

```diff
@@ -0,0 +1,24 @@
+name: Test github-dispatches
+
+on:
+  push:
+    paths:
+      - .github/workflows/github-dispatches.yaml
+      - github-dispatches/**
+  pull_request:
```

Comment:
@garethahealy So one thing to keep in mind here is that in this repository, the `pull_request` checks will always fail because this action needs access to secrets. Since `pull_requests` only have read access to repositories from forks.. they won't actually have access to the secrets.

There are workarounds for this that I'm going to try to get an example of out for, but for now it might be best to just remove the `pull_request` section from this workflow.

Reply:
@tylerauerbeck ; same as other pr, I've updated the action to skip if no creds are provided, so PRs can be built to at least get an early warning of a docker issue.